.ado/templates/setup-repo.yml

```diff
@@ -5,6 +5,11 @@ steps:
       versionSpec: '5.6.0'
     displayName: 'Install Nuget'
 
+  - task: UseNode@1
+    inputs:
+      versionSpec: '22.x'
```

Comment:
Should this be `version` instead of `versionSpec`?

https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-node-v1?view=azure-pipelines

Reply:
Thanks for catching that!